utils/sectionList.js

```diff
@@ -0,0 +1,32 @@
+import navCommunity from '../content/footer/community.yml';
+import navContact from '../content/footer/contact.yml';
+import navContribute from '../content/footer/contribute.yml';
+import navFioriService from '../content/footer/fiori-service.yml';
+import navTerms from '../content/footer/terms.yml';
```

Comment:
Generally we should look into config for the site. 
Should it go to `gatsby-config.js`

These are fine here for now as it may make sense to move them to a collection in CMS at some point.